### PR TITLE
Hotfix/1.10.1

### PR DESCRIPTION
**Pull request by NatalieEO**
Please make sure that you've checked the following before submitting your pull request. Thanks!

- [] Check that you pass the basic style checks and unit tests by running `mvn clean install`

### CODE REVIEW

File: bom-internal/pom.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@ POM as their parent.
 
     <groupId>io.dockstore</groupId>
     <artifactId>bom-internal</artifactId>
-    <version>1.11.0-alpha.0-SNAPSHOT</version>
```
Comment (denis-yuen) [inline bom-internal/pom.xml:40]: Doesn't seem right for going into develop, master should get 1.10.2-SNAPSHOT if anything, develop should keep 1.11.0-alpha.0-SNAPSHOT

File: dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java [hunk 2]
```diff
@@ -505,7 +505,7 @@ public void testUpdateUserMetadataFromGithub() {
         System.out.println(usersApi.getUser().getUserProfiles().get("github.com"));
         assertNull(userProfile.getName());
         assertEquals("dockstore.test.user2@gmail.com", userProfile.getEmail());
-        assertEquals("https://avatars1.githubusercontent.com/u/17859829?v=4", userProfile.getAvatarURL());
```
Comment (denis-yuen) [inline dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java:508]: Not sure why I'm seeing these changes, aren't these already in develop?

**Review by denis-yuen — COMMENTED** [review]
questions

File: dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java (hunk 2 above)
Comment (denis-yuen) [reply dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java:508]: Actually, this was cherry-picked

**Comment by denis-yuen** [comment]
Re-starting one build

**Review by denis-yuen — COMMENTED** [review]
Tainted now, so requesting additional reviews

File: bom-internal/pom.xml (hunk 1 above)
Comment (coverbeck) [reply bom-internal/pom.xml:40]: Agreed, but it it's still on 1.10.

File: README.md [hunk 3]
```diff
@@ -50,25 +50,6 @@ server-url = https://www.dockstore.org/api
 
 1. Ensure that you are using Java 11. Java 8 (both Open and Oracle) will not work.
 
-### Migration to Dockstore 1.3
```
Comment (coverbeck) [inline README.md:53]: Surprised this changed in the hotfix, but seems fine.

**Review by garyluu — APPROVED** [review]
Probably fine

File: bom-internal/pom.xml (hunk 1 above)
Comment (denis-yuen) [reply bom-internal/pom.xml:40]: Interesting, think the Maven versions plugin file is missing this one

File: README.md (hunk 3 above)
Comment (denis-yuen) [reply README.md:53]: This was me, testing admin push.

**Review by Richard-Hansen — APPROVED** [review]

**Review by GFJHogue — APPROVED** [review]

**Review by coverbeck — APPROVED** [review]

**Pull request merged**